amplify.yml

```diff
@@ -1,7 +1,4 @@
-version: 1
```

Comment:
Hold up: It looks like these got reverted for some reason. We'll want to revert this change before we merge